### PR TITLE
Improve poms

### PR DESCRIPTION
**Pull request by JaviVilarroig**
This is just a little cleanup in the POM files to remove some maven Warning.

### CODE REVIEW

File: vassal-app/pom.xml [hunk 1]
```diff
@@ -409,7 +408,6 @@
             <plugin>
               <groupId>org.codehaus.mojo</groupId>
               <artifactId>exec-maven-plugin</artifactId>
-              <version>3.1.0</version>
```
Comment (uckelman) [inline vassal-app/pom.xml:412]: Is this really duplicate? Does the other specification actually fix the version _here_?

Comment (JaviVilarroig) [reply vassal-app/pom.xml:412]: Yes, absolutely.

I the parent POM file, in the PluginsManagement section, line 142, We are already specifying what version of the plugin must be used in all the projects.

That's that way to allow that every module uses the same plugin versions, unless overriden.

**Comment by uckelman** [comment]
Looks good. Thanks for the contribution.

**Comment by uckelman** [comment]
I've added you to the contributors list, which will show up the next time the site is updated.

**Comment by JaviVilarroig** [comment]
Small contribution :)

Thanks!

**Pull request merged**